test/integration/node-specific/client_close.test.ts

```diff
@@ -632,7 +690,47 @@ describe.skip('MongoClient.close() Integration', () => {
 
   describe('Server resource: Cursor', () => {
     describe('after cursors are created', () => {
-      it.skip('all active server-side cursors are closed by client.close()', async function () {});
+      let client;
+      let coll;
+      let cursor;
+
+      beforeEach(async function () {
+        client = this.configuration.newClient();
+        coll = client.db('db').collection('coll');
+      });
+
+      afterEach(async function () {
+        await client?.close();
+        await cursor?.close();
+      });
+
+      it('all active server-side cursors are closed by client.close()', async function () {
+        const getCursors = async () => {
+          const res = await client
```

Comment:
optional: this is the sort of thing for which we'd usually use a utilClient.  If we use a separate client, we keep the client used for testing separate from the client we use to collect cluster data.  It also avoids reopening the client after closing it in the test (line 729).

Reply:
Made the change, yeah that's a good idea!